cdap-ui/app/directives/log-viewer/log-viewer.js

```diff
@@ -268,6 +271,7 @@ function LogViewerController ($scope, $window, LogViewerStore, myLogsApi, LOGVIE
       //otherwise, it does not have stack trace but has been selected
       vm.displayData[index].selected = !vm.displayData[index].selected;
     }
+    checkForScrollbar();
```

Comment:
Not sure what you mean, I added this change to both `log-viewer.js` and `log-viewer-preview.js`